presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/QueryBuilder.java

```diff
@@ -90,7 +91,8 @@ public PreparedStatement buildSql(
             String table,
             List<JdbcColumnHandle> columns,
             TupleDomain<ColumnHandle> tupleDomain,
-            Optional<String> additionalPredicate)
+            Optional<String> additionalPredicate,
+            Optional<Function<JdbcColumnHandle, String>> bindingPlaceholder)
```

Comment:
* No point to pass `Optional` here. You can always pass function that always return `"?"`
 * Can you make it a field of `QueryBuilder`, this function should not changed between queries, right?

Reply:
make sense. Thanks @kokosing . done